### ISSUE TITLE
Mouse not working in vscode windows terminal

### ISSUE THREAD

**Issue by hongshui3000** [Textualize/textual#1031]
The mouse can work normally under the windows terminal, but the menu cannot be clicked with the mouse under the vscode windows terminal. I am using the demo program (python -m textual)

**Comment by hongshui3000** [comment]
Windows version  : windows 10
VSCode version : 1.72.2

**Comment by davep** [comment]
For what it's worth, I've just done a simple test that didn't involve anything Textual. Picked a terminal application that responds to mouse (in this case `vim`), ran it in a normal Windows Terminal and confirmed that mouse events were processed as you'd expect. Then ran it under VSCode (yes, I did run `vim` inside VSCode!) and confirmed that mouse events were not responded to in the way you'd expect.

Conclusion being: this doesn't seem to be a Textual issue, this seems to be a Windows Terminal under VSCode under Windows 10 issue. However, we'll keep this open until we've confirmed the issue one way or another.
